src/pymovements/datasets/toy_dataset_eyelink.py

```diff
@@ -74,6 +74,12 @@ class ToyDatasetEyeLink(DatasetDefinition):
         If named groups are present in the `filename_format`, this makes it possible to cast
         specific named groups to a particular datatype.
 
+    trial_columns: list[str]
+            The name of the trial columns in the input data frame. If the list is empty or None,
+            the input data frame is assumed to contain only one trial. If the list is not empty,
+            the input data frame is assumed to contain multiple trials and the transformation
+            methods will be applied to each trial separately. (default: None)
```

Comment:
`(default: None)` doesn't make sense here since we specify it to be a `list[str]`

Reply:
you're right, was a copy-paste error

Reply:
I removed mentioning the default at the end. all dataset definition classes seem to conveniently leave the defaults out. so maybe it wasn't a copy-paste error after all.

Reply:
I changed the typehint of the field to `trial_columns: list[str] | None` to allow overwriting these without having mypy complain in my tests in #1004 

I wrote an issue for it to discuss this further #1071